HoRNDIS.cpp

```diff
@@ -1258,7 +1269,7 @@ int HoRNDIS::rndisQuery(void *buf, uint32_t oid, uint32_t in_len, void **reply,
 	u.get->len = cpu_to_le32(in_len);
 	u.get->offset = cpu_to_le32(20);
 	
-	rc = rndisCommand(u.hdr, 1025);
+	rc = rndisCommand(u.hdr, RNDIS_CMD_BUF_SZ);
```

Comment:
Good catch.  Always happy to see magic numbers removed.

Reply:
ack